OSDOCS-17171-3:CQA 2.0-Security-Certificates-Attachment4 (#115884)
openshift/openshift-docs: modules/customize-certificates-add-service-serving-apiservice.adoc

[id="add-service-certificate-apiservice_{context}"]
= Add the service CA bundle to an API service

[role="_abstract"]

> To allow the Kubernetes API server in {product-title} to validate the service CA certificate that secures an API service endpoint, you can annotate an `APIService` object to inject the service CA bundle into the `spec.caBundle` field.

Reviewer: ```suggestion
To allow the Kubernetes API server in {product-title} to validate the service Certificate Authority (CA) certificate... 
```
Define CA the first time it's used in each module.

.Procedure

. Annotate the API service with `service.beta.openshift.io/inject-cabundle=true`:
+
[source,terminal]
----

> $ oc annotate apiservice <api_service_name>

Reviewer 2: ```suggestion
$ oc annotate apiservice <api_service_name> \
```
